e2e/utils.go

```diff
@@ -23,12 +21,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/kubernetes"
-	clientset "k8s.io/client-go/kubernetes"
+
+	//clientset "k8s.io/client-go/kubernetes"
```

Comment:
don't comment it out, just remove it completely

Reply:
Its just that I didnt notice that its commented out :)..